src/Controller/AppController.php

```diff
@@ -94,16 +94,27 @@ public function initialize()
     }
 
     public function customLog($description) {
+        $user = $this->Auth->user() != null ? $this->Auth->user()['samaccountname'] : null;
+        $action = $this->request->getParam('action');
+
+        // don't log some stuff
+        if( is_null($user) ||
+            $action == 'index' ||
+            $action == 'view' ){
```

Comment:
Because I cannot remember how PHP works off the top of my head, when in regards to a string do you use double equals or triple equals? Or do you need to use a string compare/equals? (In order to deep check/not object check)

Reply:
Triple is better. Fixed.